templates/example_cell_clustering.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Same comment as for pixel clustering. Would be more helpful to have this heatmap after consensus clustering, and show a colorbar indicating the hierarchical cluster that each of the 100 clusters belongs to (see my example heatmaps). This will help the user decide whether or not they need to use Zak's tool to do any manual reclustering.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>